pkg/update/update_cockroach_version_common.go

```diff
@@ -46,39 +48,66 @@ func makeUpdateCockroachVersionFunction(
 	}
 }
 
+// TODO some of these should probably be panics or cancel the update at least
```

Comment:
I think vlad was using the context to cancel operations previously? A status seems much more explicit IMO.

Reply:
he was timing out the context after I think 5 minutes, which does not work for updates.